### PR TITLE
Getter for private validators added

### PR DESCRIPTION
**Pull request by grbIzl**

### CODE REVIEW

File: contracts/PrivateContract.sol [hunk 1]
```diff
@@ -20,6 +20,10 @@ contract PrivateContract {
         state = initialState;
         nonce = 1;
     }
+    
+    function getValidators() public constant returns (address[]) {
```
Comment (svyatonik) [inline contracts/PrivateContract.sol]: Do you want to have some kind of abstraction here? Because validators are already public.

Comment (grbIzl) [reply contracts/PrivateContract.sol]: For public fields Solidity makes only getters in form validators[index]. That's not so useful in this case

Comment (rphmeier) [reply contracts/PrivateContract.sol]: I think this requires byzantium changes to use correctly across contracts. AFAIK solidity doesn't natively support it yet.

Comment (rphmeier) [reply contracts/PrivateContract.sol]: https://github.com/ethereum/solidity/issues/3270

File: test/PrivateContractTest.js [hunk 2]
```diff
@@ -23,6 +23,21 @@ contract('Private', function(accounts) {
     private: '0x323f25528bca4eac32e75590ec62a6674240468de6ae7633f580d727642d00a6',
     address: '0xc274fcaf830aa911f1b5a32c8af21c6ee7c3d264'
   };
+  
+    describe("getValidators", () => {
+    let privateContract;
```
Comment (rphmeier) [inline test/PrivateContractTest.js]: indent?

Comment (grbIzl) [reply test/PrivateContractTest.js]: Er, IDE is not setup for such code

**Comment by debris** [comment]
this repo is being deprecated. @grbIzl please move the code of this contract to https://github.com/parity-contracts/private-tx

**Pull request closed without merging**